### PR TITLE
docs(api): modules section translated

### PR DESCRIPTION
**Pull request by raulwwq0**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Feature :sparkles:
- [ ] Code style update (formatting, renaming) :art:
- [ ] Refactoring (no functional changes, no api changes) :recycle:
- [X] Documentation content changes :memo:
- [ ] Other :zap:

## What is the new behavior?

Translate all files from the _modules_ folder.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: api/modules/pinia.md [hunk 1]
```diff
@@ -33,529 +33,512 @@ sidebarDepth: 3
...
 
-### StoreOnActionListenerContext
+### StoreOnActionListenerContext {#storeonactionlistenercontext}
 
-Ƭ **StoreOnActionListenerContext**<`Id`, `S`, `G`, `A`\>: [`_ActionsTree`](pinia.md#_actionstree) extends `A` ? [`_StoreOnActionListenerContext`](../interfaces/pinia._StoreOnActionListenerContext.md)<[`StoreGeneric`](pinia.md#storegeneric), `string`, [`_ActionsTree`](pinia.md#_actionstree)\> : { [Name in keyof A]: Name extends string ? \_StoreOnActionListenerContext<Store<Id, S, G, A\>, Name, A\> : never }[keyof `A`]
+Ƭ **StoreOnActionListenerContext**<`Id`, `S`, `G`, `A`\>: [`_ActionsTree`](pinia.md#_actionstree) extiende `A` ? [`_StoreOnActionListenerContext`](../interfaces/pinia._StoreOnActionListenerContext.md)<[`StoreGeneric`](pinia.md#storegeneric), `string`, [`_ActionsTree`](pinia.md#_actionstree)\> : { [Name en keyof A]: Name extiende string ? \_StoreOnActionListenerContext<Store<Id, S, G, A\>, Name, A\> : never }[keyof `A`]
 
-Context object passed to callbacks of `store.$onAction(context => {})`
-TODO: should have only the Id, the Store and Actions to generate the proper object
+Objecto de contexto pasado a las callbacks de `store.$onAction(context => {})`
```
Comment (edgonzalez24) [inline api/modules/pinia.md:146]: "Objeto" instance of "Objecto"

File: api/modules/pinia.md [hunk 2]
```diff
@@ -592,145 +575,142 @@ if (import.meta.hot) {
...
+Crea una función `useStore` que recupera la instancia del almacén
 
-#### Type parameters
+#### Tipado de los parámetros {#type-parameters}
 
-| Name | Type |
+| Nombre | Tipo |
 | :------ | :------ |
 | `Id` | extends `string` |
 | `S` | extends [`StateTree`](pinia.md#statetree) = {} |
```
Comment (edgonzalez24) [inline api/modules/pinia.md:626]: I noticed that elsewhere the translation of "extends" was made.

File: api/modules/pinia.md [hunk 3]
```diff
@@ -948,215 +920,203 @@ export default {
...
 
-### mapStores
+### mapStores {#mapstores}
 
 ▸ **mapStores**<`Stores`\>(...`stores`): [`_Spread`](pinia.md#_spread)<`Stores`\>
 
-Allows using stores without the composition API (`setup()`) by generating an
-object to be spread in the `computed` field of a component. It accepts a list
-of store definitions.
+Permite usar almacenes sin la API de composición (`setup()`) generando un objecto que será extendido en el campo `computed` de un componente. Acepta una lista de definiciones de almacenes.
```
Comment (edgonzalez24) [inline api/modules/pinia.md:950]: "Objeto" instance "objecto"

File: api/modules/pinia.md [hunk 4]
```diff
@@ -948,215 +920,203 @@ export default {
...
 
-#### Parameters
+#### Parámetros {#parameters}
 
-| Name | Type | Description |
+| Nombre | Tipo | Descripción |
 | :------ | :------ | :------ |
-| `useStore` | [`StoreDefinition`](../interfaces/pinia.StoreDefinition.md)<`Id`, `S`, `G`, `A`\> | store to map from |
-| `keys` | keyof `S`[] | array of state properties |
+| `useStore` | [`StoreDefinition`](../interfaces/pinia.StoreDefinition.md)<`Id`, `S`, `G`, `A`\> | almacén desde el que mappear |
```
Comment (edgonzalez24) [inline api/modules/pinia.md:1031]: "mapear" instance of "mappear"

**Comment by raulwwq0** [comment]
@edgonzalez24 everything done, can you revise it again please??

**Review by edgonzalez24 — APPROVED** [review]
LGTM 💯

**Pull request merged**